<ui version="4.0" >
 <class>TextureOffRepDlg</class>
 <widget class="QDialog" name="TextureOffRepDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit texture offsets / repeatings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="spinOffsetV" >
        <property name="maximum" >
         <double>10.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-10.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QDoubleSpinBox" name="spinOffsetU" >
        <property name="maximum" >
         <double>10.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-10.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="sliderOffsetU" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-100</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Texture offset U:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Texture offset V:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="sliderOffsetV" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-100</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinRepeatV" >
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinRepeatU" >
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Texture repeat V:</string>
        </property>
        <property name="buddy" >
         <cstring>spinRepeatV</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Texture repeat U:</string>
        </property>
        <property name="buddy" >
         <cstring>spinRepeatU</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextureOffRepDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>520</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextureOffRepDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>520</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
